triciaswift/levelup-server
# Created all models and applied all migrations

Created the following models with the following properties:

1. EventGamer
    - event - fk
    - attendee fk
2. Event
    - name char
    - date_time - datetime
    - location char
    - organizer - fk
    - game - fk
    - attendees - manytomany
3. GameType
    - label char
4. Game
    - name - char
    - manufacturer - char
    - number_of_players - int
    - type = fk
    - creator = fk

### To Test
1. Pull down the `models` branch and switch to it
2. Open VS Code
3. Open the db.sqlite3 file and make sure all the correct tables and properties are in the database.
